presto-main/src/test/java/com/facebook/presto/sql/planner/TestDomainTranslator.java

```diff
@@ -1168,8 +1175,22 @@ public void testNumericTypeTranslation()
     {
         List<NumericValues> translationChain = ImmutableList.of(
                 new NumericValues(C_DOUBLE, (value) -> doubleLiteral((double) value), -1.0 * Double.MAX_VALUE, -22.0, -44.555678, 23.0, 44.555678, Double.MAX_VALUE),
+                new NumericValues(C_DECIMAL_26_5, (value) -> decimalLiteral(Decimals.toString((Slice) value, 5)), longDecimal("-999999999999999999999.99999"), longDecimal("-22.00000"), longDecimal("-44.55568"), longDecimal("23.00000"), longDecimal("44.55567"), longDecimal("999999999999999999999.99999")),
```

Comment:
> what about DOUBLE to DECIMAL floor cast?

It is tested. `C_DOUBLE` is above the `C_DECIMAL` columns. So, it is goes down by the chain.